php-classes/Slate/CBL/CompetenciesRequestHandler.php

```diff
@@ -189,6 +189,7 @@ public static function handleDemonstrationsRequest(Competency $Competency)
             $skillDemonstration['StudentID'] = intval($skillDemonstration['StudentID']);
```

Comment:
The `\DB` class makes direct calls to the database. This is an optimized multi-table query that bypasses the ORM. I don't believe in ORM as an (leaky) abstraction layer to enable database engine portability, only a convenience layer for simple&common operations.

A call through the ORM would look like `\Slate\CBL\Demonstration::getSomethingSomehow(...)`